File: azure_mysql_connection.py
import mysql.connector
from mysql.connector import errorcode

# Obtain connection string information from the portal
config = {
  'host':'*.mysql.database.azure.com',
  'user':' ',
  'password':' ',
  'database':' '
}

# Construct connection string
try:
   conn = mysql.connector.connect(**config)
   print("Connection established")
except mysql.connector.Error as err:
  if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
    print("Something is wrong with the user name or password")
  elif err.errno == errorcode.ER_BAD_DB_ERROR:
    print("Database does not exist")
  else:
    print(err)
else:
  cursor = conn.cursor()

  #create table
  # Drop previous table of same name if one exists
  cursor.execute("DROP TABLE IF EXISTS inventory;")
  print("Finished dropping table (if existed).")
  
  # Create table
  cursor.execute("CREATE TABLE inventory (id serial PRIMARY KEY, name VARCHAR(50), quantity INTEGER);")
  print("Finished creating table.")
  
  # Insert some data into table
  cursor.execute("INSERT INTO inventory (name, quantity) VALUES (%s, %s);", ("Man", 10))
  print("Inserted",cursor.rowcount,"row(s) of data.")
  cursor.execute("INSERT INTO inventory (name, quantity) VALUES (%s, %s);", ("Woman", 15))
  print("Inserted",cursor.rowcount,"row(s) of data.")
  cursor.execute("INSERT INTO inventory (name, quantity) VALUES (%s, %s);", ("Child", 10))
  print("Inserted",cursor.rowcount,"row(s) of data.")

  # Read data
  cursor.execute("SELECT * FROM inventory;")
  rows = cursor.fetchall()
  print("Read", cursor.rowcount, "row(s) of data.")

  # Print all rows
  for row in rows:
    print("Data row = (%s, %s, %s)" % (str(row[0]), str(row[1]), str(row[2])))


  # Update a data row in the table
  cursor.execute("UPDATE inventory SET quantity = %s WHERE name = %s;", (100, "Man"))
  print("Updated",cursor.rowcount,"row(s) of data.")


  # Delete a data row in the table
  cursor.execute("DELETE FROM inventory WHERE name=%(param1)s;", {'param1':"Child"})
  print("Deleted",cursor.rowcount,"row(s) of data.")

  # Cleanup
  conn.commit()
  cursor.close()
  conn.close()
  print("Done.")

File: AzureMySQLConnection.js
const mysql = require('mysql');

function mysqlDB(){
    var config =
    {
        host: '*.mysql.database.azure.com',
        user: ' user name',
        password: 'user password ',
        database: 'database name',
        port: 3306,
        ssl: false
    };

    const conn = new mysql.createConnection(config);

    conn.connect(
        function (err) {
        if (err) {
            console.log("!!! Cannot connect !!! Error:");
            throw err;
        }
        else
        {
           console.log("Connection established.");
               queryDatabase();
        }
    });

    function queryDatabase(){
        //drop table if exists
        conn.query('DROP TABLE IF EXISTS inventory;', function (err, results, fields) {
            if (err) throw err;
            console.log('Dropped inventory table if existed.');
        })
        //create new table
        conn.query('CREATE TABLE inventory (id serial PRIMARY KEY, name VARCHAR(50), quantity INTEGER);',
            function (err, results, fields) {
                if (err) throw err;
            console.log('Created inventory table.');
        })
        //insert data into table
        conn.query('INSERT INTO inventory (name, quantity) VALUES (?, ?);', ['Mobile', 150],
            function (err, results, fields) {
                if (err) throw err;
            else console.log('Inserted ' + results.affectedRows + ' row(s).');
        })
        conn.query('INSERT INTO inventory (name, quantity) VALUES (?, ?);', ['Desktop', 154],
            function (err, results, fields) {
                if (err) throw err;
            console.log('Inserted ' + results.affectedRows + ' row(s).');
        })
        conn.query('INSERT INTO inventory (name, quantity) VALUES (?, ?);', ['Laptop', 100],
        function (err, results, fields) {
                if (err) throw err;
            console.log('Inserted ' + results.affectedRows + ' row(s).');
        })
        conn.end(function (err) {
        if (err) throw err;
        else  console.log('Done.')
        });
    };

};

mysqlDB();